www/source/tutorials/getting-started/overview.html.md

```diff
@@ -16,7 +16,7 @@ Before starting this tutorial, you need to have the following:
 *   The `hab` command-line interface tool. See [Get Habitat](/docs/get-habitat) if you don't already have this installed on your machine.
 *    An active GitHub account is recommended. If you don't already have an account, [sign up](https://github.com/) for one now. Note: This is required to upload and share your packages with others in the Habitat community.
 *   Your favorite text editor.
-*   If you are running Mac OS X on your host machine, then you need [Docker for Mac](https://www.docker.com/products/docker) installed. Make sure you have a Docker machine running before proceeding through the tutorial. 
+*   If you are running Mac OS X on your host machine, then you need [Docker for Mac](https://www.docker.com/products/docker#/mac) installed, and if you are running Microsoft Windows, you will need [Docker for Windows](https://www.docker.com/products/docker#/windows) installed. Make sure you have a Docker machine running before proceeding through the tutorial. 
```

Comment:
I actually think this reads better if is formatted like so:

`*  If you are running Mac OS X`
`*  If you are running Microsoft Windows`

as separately line items, tis a minor nit though really.